### DISCUSSION TITLE
How to use the fourth parameter in VLOOKUP formula?

### DISCUSSION THREAD

**Discussion by nathipg** [handsontable/hyperformula#867] [General]
Hello!

I'm trying to use the boolean param in VLOOKUP formula, but I'm having a bad time passing it.

This is what I tried to do:

https://jsfiddle.net/57fgkh0r/2/

In this sample, I tried to pass the fourth param in three diferent ways, but none of then work.

What is the right way of doing it?

**Comment by nathipg** [comment]
Actually the right way is using double quotes. Sample updated.

**Comment by bardek8** [comment]
Did you try with the function?
```
['Apples', '=VLOOKUP(A5, A1:B3, 2, FALSE())'],
['Apples', '=VLOOKUP(A5, A1:B3, 2, TRUE())'],
```
